cdap-ui/app/cdap/components/Wizard/index.js

```diff
@@ -135,6 +138,38 @@ export default class Wizard extends Component {
     return state && state.__complete;
   }
 
+  handleCallToActionClick() {
+    this.eventEmitter.emit(globalEvents.CLOSEMARKET);
+  }
+
+  handleMainCallToActionClick() {
+    if (this.state.callToActionInfo.buttonOnClick) {
```

Comment:
We should declare `callToActionInfo` in the state in constructor. Just to make sure it doesn't error out as js error if `callToActionInfo` is undefined.